src/cases/models/case-document.js

```diff
@@ -14,7 +14,8 @@ export class CaseDocument {
     this.currentStage = props.currentStage;
     this.timeline = props.timeline;
     this.stages = props.stages;
-    this.timeline = props.timeline;
+    this.requiredRoles = props.requiredRoles;
+    this.pages = props.pages;
```

Comment:
`pages` and `requiredRoles` are not persisted as part of a case but `timeline` is